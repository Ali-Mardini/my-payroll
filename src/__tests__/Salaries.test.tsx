import { describe, expect, test } from "vitest";
import { MemoryRouter } from "react-router-dom";
import { ToastContainer } from "react-toastify";
import { render } from '@testing-library/react';
import SalarieseWithAuthentication from '../pages/salaries/salaries';

const renderSalariesPage = () =>{
	const { container } = render(
	<MemoryRouter>
        <ToastContainer />
        <SalarieseWithAuthentication />
	</MemoryRouter>
    );

	return container;
};

describe("Test ==>  Salaries page", () => {
  test("renders correctly", () => {
    const container  = renderSalariesPage();
    expect(container).toMatchSnapshot();
  });
});