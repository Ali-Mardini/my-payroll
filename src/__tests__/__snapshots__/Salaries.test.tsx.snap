// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test ==>  Dashboard page > renders correctly 1`] = `
<div>
  <div
    class="Toastify"
  />
  <div
    class="flex justify-center items-center h-screen"
  >
    <div
      class="animate-spin rounded-full h-32 w-32 border-t-4 border-b-4 border-blue-500"
    />
  </div>
</div>
`;

exports[`Test ==>  Salaries page > renders correctly 1`] = `
<div>
  <div
    class="Toastify"
  />
  <div
    class="flex justify-center items-center h-screen"
  >
    <div
      class="animate-spin rounded-full h-32 w-32 border-t-4 border-b-4 border-blue-500"
    />
  </div>
</div>
`;
